buildscripts/templates/registry/incubating_java/weaver.yaml

```diff
@@ -19,12 +19,11 @@ templates:
     filter: > 
       semconv_grouped_attributes({
         "exclude_root_namespace": $excluded_namespaces,
-        "exclude_stability": [],
-        "exclude_deprecated": false
+        "exclude_stability": []
       }) | map({
           root_namespace: .root_namespace,
           attributes: .attributes,
-          excluded_attributes: $excluded_attributes[]
+          excluded_attributes: $excluded_attributes
```

Comment:
without this it was only working with a single item in the excluded_attributes list